elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -134,7 +136,10 @@ def render(element_html, data):
             else:
                 a_sub = phs.json_to_sympy(a_sub, allow_complex=allow_complex)
             a_sub = a_sub.subs(sympy.I, sympy.Symbol(imaginary_unit))
-            html_params['a_sub'] = sympy.latex(a_sub)
+            if data['raw_submitted_answers'].get(name, '').strip() == '':
+                html_params['a_sub'] = '\\texttt{(blank)}'
```

Comment:
Same comment here. I might be misunderstanding, but I think that pl-number-input and pl-matrix-input will both show the `blank-value` in the submission (if it's used), whereas pl-integer-input and pl-symbolic-input will show `blank`. I prefer the former behavior, because it lets the student see what's happening.